pandas/tests/copy_view/test_chained_assignment_deprecation.py

```diff
@@ -66,14 +67,72 @@ def test_methods_iloc_getitem_item_cache(func, args, using_copy_on_write):
     ser = df["a"]
     getattr(ser, func)(*args, inplace=True)
 
+    df = df_orig.copy()
+    df["a"]  # populate the item_cache
+    # TODO(CoW-warn) because of the usage of *args, this doesn't warn on Py3.11+
+    if using_copy_on_write:
+        with tm.raises_chained_assignment_error(not PY311):
+            getattr(df["a"], func)(*args, inplace=True)
```

Comment:
I updated the first test to also use `getattr(..)(*args)` for this part of the test, which means we are for now asserting this incorrectly doesn't raise a warning for 3.11+ because of the different ref count when using `*args`